commands/mute.js

```diff
@@ -0,0 +1,54 @@
+const {roleIDs} = require('../config.json');
+
+module.exports = {
+	name: 'mute',
+    description: 'Muting users',
+    usage: 'mute',
+	execute(message, args) {
+        const fs = {readFileSync, writeFile} = require ('fs');
```

Comment:
const {readFileSync, writeFile} = require ('fs');

Reply:
That wasn't working.